src/main.cpp

```diff
@@ -0,0 +1,5 @@
+#include "ConfigManager.h"
+
+void setup() { }
```

Comment:
This file isnt needed. Use the `simple` demo instead.

Reply:
Oh, sweet, I didn't know you could do that :)